test/org/javarosa/xform/parse/XFormParserTest.java

```diff
@@ -387,45 +386,6 @@ public void parseFormWithSetValueAction() throws IOException {
         assertThat(groupElement.getBind(), is(expectedXPathReference));
     }
 
-    /**
-     * Attributes that started being used by clients without being added as fields to DataBinding or QuestionDef should
-     * be passed through and made available in the bindAttributes or additionalAttributes list.
-     */
-    @Test public void passedThroughAttributesHaveExpectedValues() throws IOException {
-        ParseResult parseResult = parse(r("whitelisted-attributes.xml"));
```

Comment:
The XML file should be unused and removable.

Reply:
Yas you were right, I removed that file.